packages/manager/apps/public-cloud/src/index.routes.js

```diff
@@ -1,27 +1,43 @@
+import isEmpty from 'lodash/isEmpty';
+
 import { DEFAULT_PROJECT_KEY } from './index.constants';
 
 export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
   $stateProvider.state('app', {
     url: '/?onboarding',
-    redirectTo: (trans) =>
-      trans
-        .injector()
-        .get('publicCloud')
-        .getDefaultProject()
-        .then((projectId) =>
-          projectId
-            ? {
-                state: 'pci.projects.project',
-                params: {
-                  projectId,
-                },
-              }
-            : {
-                state: trans.params().onboarding
-                  ? 'pci.projects.onboarding'
-                  : 'pci.projects.new',
+    redirectTo: (trans) => {
+      const $q = trans.injector().get('$q');
+      const publicCloud = trans.injector().get('publicCloud');
+      return $q
+        .all([
+          publicCloud.getDefaultProject(),
+          publicCloud.getProjects([
+            {
+              field: 'status',
+              comparator: 'like',
+              reference: 'suspended',
+            },
+          ]),
+        ])
+        .then(([defaultProjectId, suspendedProjects]) => {
+          if (Array.isArray(suspendedProjects) && !isEmpty(suspendedProjects)) {
+            return { state: 'pci.projects' };
+          }
```

Comment:
If the customer have a default project but also suspended projects, we redirect to projects list ? Is it what we want ? 
Because the default project is here to redirect the customer by default to his project... 🤔

Reply:
Yes... This is the requirement... It would be easier to bring any suspended project, to the notice of the user. When there is no suspended project, we still redirect to the default project